registration/models.py

```diff
@@ -2,6 +2,7 @@
 from django.db import models
 from .validators import validate_family_parent, validate_student, validate_mc_options
 from django.contrib.postgres.fields import ArrayField
+from .validators import validate_family_parent, validate_student, validate_interactions
```

Comment:
this might be from a weird merge? would be good to keep all the imports from .validators on one line :)